client/src/plugins/report-feedback/ReportFeedbackOverlay.js

```diff
@@ -27,7 +27,6 @@ export function ReportFeedbackOverlay(props) {
     <Overlay
       anchor={ props.anchor }
       onClose={ props.onClose }
-      maxHeight="calc(100vh - 250px)"
```

Comment:
**Explanation:** this was causing the the report feedback overlay to shorten and have scroll, even though there was plenty of space still. The height of this overlay is a lot smaller than `MIN_HEIGHT` of the Modeler, meaning we don't need to set this.